### DISCUSSION TITLE
Could not compile query, because the Python bridge has not been initalized

### DISCUSSION THREAD

**Discussion by Benjamin-OPC** [AltimateAI/vscode-dbt-power-user#235] [Q&A]
When trying to compile code in a model, I get this message 

`Could not compile query, because the Python bridge has not been initalized. If the issue persists, please open a Github issue.`

My python virtual env is well defined, dbt is installed inside it, and the right interpreter is selected in VSCode.

Can anyone please help me ?

**Comment by mdesmet** [comment]
In general it means that your project can't be parsed. There should be another error before. Can you share that error?

**Reply by Benjamin-OPC** [reply]
Hi @mdesmet,

If I run a dbt compile or dbt run command from the vscode terminal, it works fine. No error as well if I run dbt debug 

I downgraded to 0.8.7 and it works now....

**Reply by mdesmet** [reply]
Can you provide me with a reproduction repo?

You could also check the console log in Help > Toggle Developer tools and see if there is anything fishy there.

**Comment by nemtsinkow2** [comment]
Check you Python: Selected Interpreter. (Solved it for me.)

**Comment by niteshsharoff** [comment]
I've found if I'm using environment variables and I don't have a `.env` file created with them (many times in my venv I run a `.bat` script at set them, I get this warning message - if I create my `.env` all is good.
